README.md

```diff
@@ -1,57 +1,16 @@
 # AAI-501
 
-## Super Cool Project
+## Brain Tumor Detection Project Overview
 
-### Contributors:
-Dimitri Dumont
-
-Gurleen Virk
-
-Mythreyi
```

Comment:
Please add full name : Mythreyi Thirumalai

Reply:
Do we want to keep TrainV2.py , Training.py and train.ipynb in the finalized one?

Reply:
I don't see the requirements.txt being updated 
I think there were some libraries missing there - aggregating from Gurleen's code too. 
pillow
yaml
cv2
keras
flask
uvicorn

Reply:
Gotchya!